src/loaders/abk_load.c

```diff
@@ -435,6 +437,16 @@ static int abk_test(HIO_HANDLE *f, char *t, const int start)
         return -1;
     }
 
+    /* Attempt to read title. */
+    hio_read32b(f); /* instruments_offset */
+    song_section_offset = hio_read32b(f);
+
+    if (t != NULL && read_abk_song(f, &song, AMOS_MAIN_HEADER + song_section_offset) == 0)
+    {
+        memcpy(t, song.song_name, AMOS_STRING_LEN);
```

Comment:
took me some hunting in the code, but looks OK.

Should you run libxmp_copy_adjust() on the name?

Reply:
That's probably a better idea, yes—will patch soon.